tap/runner.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Can you remove the #! line? This file is not an executable and there's not much purpose to having one in a module.